tests/db/conftest.py

```diff
@@ -60,11 +59,12 @@ def cleanup_tables(request, node_config):
     def fin():
         conn = connect()
         try:
-            conn.run(r.db(db_name).table('bigchain').delete())
-            conn.run(r.db(db_name).table('backlog').delete())
-            conn.run(r.db(db_name).table('votes').delete())
-        except r.ReqlOpFailedError as e:
-            if e.message != 'Database `{}` does not exist.'.format(db_name):
+            schema.drop_database(conn, db_name)
+            schema.create_database(conn, db_name)
+            schema.create_tables(conn, db_name)
+            schema.create_indexes(conn, db_name)
```

Comment:
No, that's the annoying thing -- in the RDB specific verison, you could do that, but since the API doesn't expose dropping tables (just the whole db) you have to set it up from scratch each time.